src/cromshell/submit/command.py

```diff
@@ -0,0 +1,131 @@
+import click
+import logging
+
+import json
+import requests
+import csv
+import shutil
+import subprocess
+from pathlib import Path
+from pathlib import PurePath
+from datetime import datetime
+
+from cromshell.utilities import cromshellconfig
+from cromshell.utilities import http_utils
+from cromshell.utilities import io_utils
+from cromshell import log
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="submit")
+@click.argument("wdl", type=click.Path(exists=True),  required=True)
+@click.argument("wdl_json", type=click.Path(exists=True),  required=True)
+@click.argument("options_json", type=click.Path(exists=True),  required=False)
+@click.argument("dependencies_zip", type=click.Path(exists=True),  required=False)
+@click.pass_obj
+def main(config, wdl, wdl_json, options_json, dependencies_zip):
+    """Submit a workflow and arguments to the Cromwell Server"""
+
+    LOGGER.info("submit")
+
+    io_utils.assert_required_file_is_not_empty(wdl, "WDL")
+    io_utils.assert_required_file_is_not_empty(wdl_json, "Input JSON")
+    io_utils.assert_required_file_is_not_empty(options_json, "Options json")
+    io_utils.assert_required_file_is_not_empty(dependencies_zip, "Dependencies Zip")
+
+    # At this point, we should validate our inputs if we can:
+    womtool_validate_wdl_and_json(wdl, wdl_json)
+
+    # Check if Cromwell Server Backend works
+    http_utils.assert_can_communicate_with_server(config)
+
+    # Submit workflow to Cromwell Server
+    LOGGER.info(f"Submitting job to server: {cromshellconfig.cromwell_server}")
+    submission_params = {
+        'workflowSource': open(wdl, 'rb'),
+        'workflowInputs': open(wdl_json, 'rb')
+    }
+    if options_json is not None:
+        submission_params['workflowOptions'] = open(options_json, 'rb')
+    if dependencies_zip is not None:
+        submission_params['workflowDependencies'] = open(dependencies_zip, 'rb')
```

Comment:
this is a bit dangerous, as if anything happens between here and end of function, these resources are not closed. 
So please follow [this](https://stackoverflow.com/questions/4617034/how-can-i-open-multiple-files-using-with-open-in-python).

Reply:
done

Reply:
@SHuang-Broad 
I get the following from `requests`
```
    fdata = fp.read()
ValueError: read of closed file
```
I think `requests` needs the files to be opened in order to send them to the server. I'll just add in closing statements after the `requests` line

Reply:
Also, I don't think it would be possible to make one long `with` statement because options and dependencies might be `None`

Reply:
Hmm, there should be a safe way to do this.
At the minimum it should be wrapped in a try-catch block to prevent leak.
We can talk " in person" later.